trade-finance-manager-api/src/v1/validation/validate-payloads.ts

```diff
@@ -0,0 +1,34 @@
+import { asString } from '../../utils/validation';
+import { isValidMongoId } from './validateIds';
+
+export const validateUpdateUtilisationReportPayloadReport = (report: object) => {
+  if ('id' in report) {
+    const isValidReportId = isValidMongoId(asString(report.id)) && Object.keys(report).length === 1;
+    if (!isValidReportId) {
+      throw new Error("'report.id' must be a valid mongo id string");
+    }
+    return true;
+  }
+
+  const reportContainsBankId = 'bankId' in report;
+  const reportContainsMonth = 'month' in report;
+  const reportContainsYear = 'year' in report;
+  const reportContainsExtraKeys = Object.keys(report).length !== 3;
```

Comment:
like wise here, we either have a valid report ID, or we have the valid other combination and we can just get rid of anything else in the body